### ISSUE TITLE
Add less than or equal to (lte) | greater than or equal to (gte)

### ISSUE THREAD

**Issue by kristianserrano** [kgar/foundry-vtt-more-handlebars-helpers#11]
- Less than or equal to: `lte`
- Greater than or equal to: `gte`

**Comment by kgar** [comment]
Consider it done.

Now, the question is... 

Approach A:
```handlebars
{{#more-handlebars-helpers-gte a b }}
  Yes
{{else}}
  No
{{/more-handlebars-helpers-gte}}
```

Or Approach B:
```handlebars
{{#if (more-handlebars-helpers-gte a b) }}
  Yes
{{else}}
  No
{{/if}
```

**Comment by kristianserrano** [comment]
Thanks! I feel like B is more consistent with existing Foundry VTT helpers.

**Comment by kgar** [comment]
Thank you for your patience. I have released this in 1.4.1.
